app/experimenter/nimbus-ui/src/components/FormBranches/FormBranch.test.tsx

```diff
@@ -0,0 +1,153 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+
+import { MOCK_CONFIG } from "../../lib/mocks";
+import {
+  SubjectBranch,
+  MOCK_BRANCH,
+  MOCK_FEATURE_CONFIG,
+  MOCK_FEATURE_CONFIG_WITH_SCHEMA,
+} from "./mocks";
+
+describe("FormBranch", () => {
+  it("renders as expected", async () => {
+    render(<SubjectBranch />);
+    expect(screen.getByTestId("FormBranch")).toBeInTheDocument();
+    expect(screen.queryByTestId("control-pill")).not.toBeInTheDocument();
+    expect(screen.queryByTestId("equal-ratio")).not.toBeInTheDocument();
+    expect(screen.queryByTestId("feature-config-edit")).toBeInTheDocument();
+    expect(screen.queryByTestId("feature-config-add")).not.toBeInTheDocument();
+    expect(screen.queryByTestId("feature-value-edit")).not.toBeInTheDocument();
+  });
+
+  it("includes a control label when reference branch", () => {
+    render(<SubjectBranch isReference />);
+    expect(screen.getByTestId("control-pill")).toBeInTheDocument();
+  });
+
+  it("indicates equal ratio when enabled", () => {
+    render(<SubjectBranch equalRatio />);
+    expect(screen.getByTestId("equal-ratio")).toBeInTheDocument();
+  });
+
+  xit("reflects when feature is disabled", () => {
```

Comment:
is this coming back or should it be removed?

Reply:
Ah yeah, that shouldn't be disabled.